cmd/utils/flags.go

```diff
@@ -1490,7 +1488,7 @@ func MakeChain(ctx *cli.Context, stack *node.Node) (chain *core.BlockChain, chai
 	var err error
 	chainDb = MakeChainDatabase(ctx, stack)
 	config, _, err := core.SetupGenesisBlock(chainDb, MakeGenesis(ctx))
-	if ctx.GlobalString(SyncModeFlag.Name) == "celolatest" {
+	if ctx.GlobalString(SyncModeFlag.Name) == "ultralight" {
```

Comment:
This is weird, should this have originally been checking if the syncmode was "ultralight" *or* "celolatest"?

If we weren't setting this config variable before for `ultralight`, perhaps it's not necessary now that `celolatest` is deprecated?

Reply:
Yeah. It is a bug. It does not impact the normal execution *but* execution of specific functions - `importChain`, `exportChain`, `importPreimages`, `exportPreimages`, and `dump`. And in all those cases, when we start Geth, `fullHeaderChainAvailable` will get the correct value since it is not being filled from this code path.